app.js

```diff
@@ -24,143 +24,160 @@ SurveyItem.prototype.buildSurveyItem = function(conatainerId){
-var chair2 = new SurveyItem('Leah2','$99.99','Neimoidia bollux momaw droid saffa sing b4-d4 tierce. Saurin roan jarael kastolar nunb ralter.','images/chair.jpg');
-var scissors2 = new SurveyItem('Han2','$7.89','Shistavanen phlog depa oppo qui-gon t88 dorvalla. Ailyn darth irek max jabba finis kiffar antonio bith.','images/scissors.jpg');
-var water_can2 = new SurveyItem('Boba2','$10.00','Quence mirta kalee qui-gon halla seerdon bibble. Calamari jacen tund tierce hutt salacious alderaan.','images/water_can.jpg');
+SurveyItem.prototype.setShownValues = function () {
+  this.numberOfTimesShown++;
+  this.usedInLastItemSet = true;
+};
+
+var itemObjects = [
+  new SurveyItem('Jabba','$19.50','Ubb maul mace gev jerjerrod dressellian. Falleen arkanis iridonian desann skywalker greedo priapulin hapes.','images/boots.jpg')
+  ,new SurveyItem('Leah','$99.99','Neimoidia bollux momaw droid saffa sing b4-d4 tierce. Saurin roan jarael kastolar nunb ralter.','images/chair.jpg')
+  ,new SurveyItem('Han','$7.89','Shistavanen phlog depa oppo qui-gon t88 dorvalla. Ailyn darth irek max jabba finis kiffar antonio bith.','images/scissors.jpg')
+  ,new SurveyItem('Boba','$10.00','Quence mirta kalee qui-gon halla seerdon bibble. Calamari jacen tund tierce hutt salacious alderaan.','images/water_can.jpg')
+  ,new SurveyItem('Vader','$23.99','Vader saffa arkanis kyle givin yavin. Utapaun selkath dat mccool zuggs medon derlin kathol secura.','images/wine_glass.jpg')
+  ,new SurveyItem('Jabba2','$19.50','Ubb maul mace gev jerjerrod dressellian. Falleen arkanis iridonian desann skywalker greedo priapulin hapes.','images/boots.jpg')
+  ,new SurveyItem('Leah2','$99.99','Neimoidia bollux momaw droid saffa sing b4-d4 tierce. Saurin roan jarael kastolar nunb ralter.','images/chair.jpg')
+  ,new SurveyItem('Han2','$7.89','Shistavanen phlog depa oppo qui-gon t88 dorvalla. Ailyn darth irek max jabba finis kiffar antonio bith.','images/scissors.jpg')
+  ,new SurveyItem('Boba2','$10.00','Quence mirta kalee qui-gon halla seerdon bibble. Calamari jacen tund tierce hutt salacious alderaan.','images/water_can.jpg')
+];
 
-var itemObjects = [boots,chair,scissors,water_can,wine_glass,boots2,chair2,scissors2,water_can2];
 var itemsObjectsWorking;
+var itemsChosen = 0;
 
-function getFirstRandom(){
+function displaySingleRandom(elementIndex,locationID){
   var index = Math.floor(Math.random() * (itemsObjectsWorking.length));
-  if(document.getElementsByClassName('surveyImageBox')[0] && document.getElementsByClassName('surveyItemDesc')[0]){
-    var oldImage = document.getElementsByClassName('surveyImageBox')[0];
-    var oldItemDesc = document.getElementsByClassName('surveyItemDesc')[0];
+  if(document.getElementsByClassName('surveyImageBox')[elementIndex] && document.getElementsByClassName('surveyItemDesc')[elementIndex]){ //remove old elements if the exist
```

Comment:
Performance-wise, it's best to avoid repeating DOM lookups if you can help it. Especially since the code that follows already assigns them to variables. I'd probably move this lookup up out of `displaySingleRandom()`, but if it were left here I would write it like this:

```javascript
var oldImage = document.getElementsByClassName('surveyImageBox')[elementIndex];
var oldItemDesc = document.getElementsByClassName('surveyItemDesc')[elementIndex];

oldImage && oldImage.remove();
oldItemDesc && oldItemDesc.remove();
```